protos/telemetry/telemetry.proto

```diff
@@ -577,7 +577,11 @@ enum FixType {
 message Battery {
     uint32 id = 3 [(mavsdk.options.default_value)="0"]; // Battery ID, for systems with multiple batteries
     float voltage_v = 1 [(mavsdk.options.default_value)="NaN"]; // Voltage in volts
-    float remaining_percent = 2 [(mavsdk.options.default_value)="NaN"]; // Estimated battery remaining (range: 0.0 to 1.0)
+    float remaining_percent = 2 [(mavsdk.options.default_value)="NaN"]; // Estimated battery remaining (range: 0 to 100)
+    float current_battery_a = 4 [(mavsdk.options.default_value)="NaN"]; // Battery current, NAN if autopilot does not measure the current
+    float current_consumed_ah = 5 [(mavsdk.options.default_value)="NaN"]; // Consumed charge, NAN if autopilot does not provide consumption estimate
+    float energy_consumed_j = 6 [(mavsdk.options.default_value)="NaN"]; // Consumed energy, NAN if autopilot does not provide energy consumption estimate
+    float temperature_degc = 7 [(mavsdk.options.default_value)="NaN"]; // Temperature of the battery. NAN for unknown temperature
```

Comment:
```suggestion
    float current_battery_a = 4 [(mavsdk.options.default_value)="NaN"]; // Battery current in Amps, NAN if autopilot does not measure the current
    float current_consumed_ah = 5 [(mavsdk.options.default_value)="NaN"]; // Consumed charge in Amp hours, NAN if autopilot does not provide consumption estimate
    float energy_consumed_wh = 6 [(mavsdk.options.default_value)="NaN"]; // Consumed energy in Watt hours, NAN if autopilot does not provide energy consumption estimate
    float temperature_degc = 7 [(mavsdk.options.default_value)="NaN"]; // Temperature of the battery in degrees Celsius. NAN for unknown temperature
```

What about that?

Reply:
Thank you for your suggestion, I did the changes